packages/ricos-content-query/web/src/editor-query/queries/is-text-block-node.ts

```diff
@@ -0,0 +1,11 @@
+import type { QueryItem } from 'ricos-types';
+
+const NAME = 'isTextBlockNode';
+const isTextBlockNode: QueryItem =
+  ({ editorQuery }) =>
+  () => {
+    const selectedNode = editorQuery.selection().doc.children[0];
```

Comment:
what if the doc has no children?
```suggestion
    const selectedNode = editorQuery.selection().doc?.children?.[0];
```